package/MDAnalysis/analysis/rdf_s.py

```diff
@@ -0,0 +1,166 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+"""
+Radial Distribution Functions --- :mod:`MDAnalysis.analysis.rdf`
+================================================================
+
+Tools for calculating pair distribution functions ("radial
+distribution functions" or "RDF").
+
+.. Not Implemented yet:
+.. - Structure factor?
+.. - Coordination number
+
+"""
+from __future__ import division, absolute_import
+import numpy as np
+
+from ..lib.util import blocks_of
+from ..lib import distances
+from .base import AnalysisBase
+from six.moves import zip, range
+
+
+class InterRDF_s(AnalysisBase):
+    """Intermolecular pair distribution function
+
+    InterRDF_s(u, g1, g2, nbins=75, range=(0.0, 15.0))
+
+    Arguments
+    ---------
+    u : Universe
+       A Universe contains atoms in ags
+    ags : list
+         A list AtomGroup pairs
+    nbins : int (optional)
+          Number of bins in the histogram [75]
+    range : tuple or list (optional)
+          The size of the RDF [0.0, 15.0]
+    start : int (optional)
+          The frame to start at (default is first)
+    stop : int (optional)
+          The frame to end at (default is last)
+    step : int (optional)
+          The step size through the trajectory in frames (default is
+          every frame)
+
+    Example
+    -------
+    First create the :class:`InterRDF_s` object, by supplying one Universe
+    and two AtomGroup selection lists then use the :meth:`run` method ::
```

Comment:
Change: you supply `ag`, which is a list of pairs of AtomGroups.

Reply:
Make your example more verbose. Show how you would select the AtomGroups and how you would create `ag`. You can use what you put in the tests. You can even use the tests:
```python
from MDAnalysis.tests.datafiles import GRO_MEMPROT, XTC_MEMPROT
u = mda.Universe(GRO_MEMPROT, XTC_MEMPROT)
s1 = u.select_atoms('name ZND and resid 289')
s2 = u.select_atoms('(name OD1 or name OD2) and resid 51 and sphzone 5.0 (resid 289)')
s3 = u.select_atoms('name ZND and (resid 291 or resid 292)')
s4 = u.select_atoms('(name OD1 or name OD2) and sphzone 5.0 (resid 291)')
ag = [[s1, s2], [s3, s4]]

rdf = InterRDF_s(u. ag)
rdf.run()
```